src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const getBody = document.querySelector('body');
+
+  const createElementDiv = document.createElement('div');
+
+  createElementDiv.classList.add('notification');
+  createElementDiv.classList.add(type);
+
+  createElementDiv.innerHTML = `
+    <h2 class="title">${title}</h2>
+    <p>${description}</p>
+  `;
+
+  createElementDiv.style.top = `${posTop}px`;
+  createElementDiv.style.right = `${posRight}px`;
+
+  getBody.appendChild(createElementDiv);
+
+  // setTimeout(() => createElementDiv.remove(), 2000);
```

Comment:
You need to uncomment the timeout and also you don't need to delete the message completely, use display instead
```suggestion
  setTimeout(() => {
    createElementDiv.style.display = 'none';
  }, 2000);
```